src/mumble/UserLocalNicknameDialog.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_USERNICKNAME_H_
+#define MUMBLE_MUMBLE_USERNICKNAME_H_
+
+#include <memory>

```

Comment:
I think this header should also include `<unordered_map>` to make it self-contained